docs/content/doc/developers/integrations.en-us.md

```diff
@@ -24,3 +24,20 @@ If you are looking for [CI/CD](https://gitea.com/gitea/awesome-gitea#user-conten
 an [SDK](https://gitea.com/gitea/awesome-gitea#user-content-sdk),
 or even some extra [themes](https://gitea.com/gitea/awesome-gitea#user-content-themes),
 you can find them listed in the [awesome-gitea](https://gitea.com/gitea/awesome-gitea) repository!
+
+## Create a New File
+
+If you'd like to create a new file with a given name and contents,
+you can do so with query parameters:
+
+```txt
+GET /{{org}}/{{repo}}/_new/{{filepath}}?filename={{filename}}&value={{content}}
```

Comment:
Why is this not `POST`?

Reply:
My guess is because this is moreso for "suggesting" name and contents, similar to our issue query params for title/body

Reply:
See the short demo. This is for pre-filling the contents the same as GitHub (and soon Gist) do, so that you can use tools like [Bliss](https://github.com/coolaj86/bliss) to quickly create content from templates.

<img width="1053" alt="Screen Shot 2021-07-25 at 3 00 04 AM" src="https://user-images.githubusercontent.com/122831/127208327-e2a084b2-a9c3-4fd0-a4f3-a02c3b3e046c.png">

Reply:
Ah, ok, so it's not actually sending data. I guess GET is fine in that case.

Reply:
Well, it's literally what already exists. I didn't add something new here. I just made it more capable of doing what it already does.